templates/store/categories/iot.html

```diff
@@ -53,16 +53,24 @@ <h3 class="p-heading--four">Industrial IoT</h3>
       {% endfor %}
     </div>
 
-    <div class="u-fixed-width">
-      <div class="p-card--highlighted">
-        <div class="row u-no-padding--left u-no-padding--right">
-          <div class="col-6">
+    <div class="row u-equal-height">
```

Comment:
I don't think you need `u-equal-height` on the `row` element as well. And in theory should have equal-height by default, is that not right @lyubomir-popov ?